localization/english/wc_character_triggers_l_english.yml

```diff
@@ -0,0 +1,7 @@
+﻿l_english:
+ give_magic_eyes_interaction:0 "Give Magic Eyes"
+
+ stasis_trap_already_stunned_first_not:0 "I am #bold not#! stunned"
+ stasis_trap_already_stunned_first:0 "I am stunned"
+ stasis_trap_already_stunned_third_not:0 "[CHARACTER.GetShortUIName] is #bold not#! stunned"
+ stasis_trap_already_stunned_third:0 "[CHARACTER.GetShortUIName] is already stunned"
```

Comment:
Huh? New file? There's a file with the same name in `Warcraft-Guardians-of-Azeroth-2\localization\english\triggers`.